docs/install.rst

```diff
@@ -77,9 +77,9 @@ you need these libraries.
 Get and run Read the Docs
 -------------------------
 
-Clone the repository somewhere on your disk and enter to the repository::
+Clone the repository somewhere on your disk, initialize and update the submodule, and enter to the repository::
```

Comment:
We can remove this sentence now I think, this is done implicitly.

Reply:
(the sentence about submodules only)

Reply:
Done!